packages/upset/src/utils/styles.ts

```diff
@@ -25,3 +25,7 @@ export const queryColorPalette = [
   '#bcbd22',
   '#17becf',
 ];
+
+export const mousePointer = css`
+  cursor: pointer;
+`;
```

Comment:
This needs a newline. Let's open an issue to add the linter, since these kind of changes should be automatic

Reply:
#129 issue created